packages/devtools_app/lib/src/framework/landing_screen.dart

```diff
@@ -352,6 +359,46 @@ class AppSizeToolingInstructions extends StatelessWidget {
   }
 }
 
+@visibleForTesting
+class SnapshotAnalysisInstructions extends StatelessWidget {
+  const SnapshotAnalysisInstructions({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+    return LandingScreenSection(
+      title: 'Snapshot Analysis',
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            'Analyze and diff the saved memory snapshots',
+            style: textTheme.titleMedium,
+          ),
+          const SizedBox(height: denseRowSpacing),
+          Text(
+            'Analyse memory heap snapshots, previously saved with DevTools or Dart leak_tracker.',
```

Comment:
sp: Analyze

"heap snapshots that were previously saved from DevTools or package:leak_tracker"

and making 'package:leak_tracker' a hyperlink here would be good.